ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -117,28 +123,47 @@ public class BadgeViewTokenSet: ControlTokenSet<BadgeViewTokenSet.Tokens> {
                     switch style() {
                     case .default:
                         return theme.color(.brandForegroundDisabled1)
-                    case .danger, .severeWarning, .warning, .success, .neutral:
+                    case .danger, .severe, .warning, .success, .neutral:
                         return theme.color(.foregroundDisabled1)
                     }
                 }
+            case .strokeTintColor:
+                return .uiColor {
+                    switch style() {
+                    case .default:
+                        return theme.color(.brandForegroundTint)
+                    case .danger:
+                        return theme.color(.dangerStroke1)
+                    case .severe:
+                        return theme.color(.severeStroke1)
+                    case .warning:
+                        return theme.color(.warningStroke1)
```

Comment:
Sorry, probably should've caught this last time, but I guess seeing it in context helps. Our definition of Yellow Shade 40 is wrong, we have 0x4C4400 and it should be 0x81740

Reply:
Or possibly the design was supposed to be shade 30.